pkg/channel/lookup.go

```diff
@@ -45,6 +46,11 @@ func GetTemplates(
 		switch {
 		case module.RemoteModuleTemplateRef == "":
 			template = NewTemplateLookup(kymaClient, module, kyma.Spec.Channel).WithContext(ctx)
+			if template.Err == nil {
```

Comment:
coding style issue:
```
if template.Err != nil {
  continue
}
if err := saveDescriptorToCache(template); err != nil {
  template.Err = fmt.Errorf("failed to get descriptor: %w", err)
}
```

Reply:
I still need to reach this line `templates[module.Name] = &template` not to continue in this case. That's why I have it like this